specifyweb/frontend/js_src/lib/components/HomePage/taxonTileHelpers.ts

```diff
@@ -132,7 +125,20 @@ export function makeTreeMap(container: SVGElement, rawRoot: PairedNode) {
     .size([container.clientWidth, container.clientHeight])
     .round(true)(root);
 
-  const color = scaleOrdinal(schemeSet2);
+  const tileColors = [
```

Comment:
(optional) usually we define static arrays like this outside the function (idally right below or right above it)

reasons:
- it's a common convention in programming that config variables/options are at the top level of the file (top level does mean top of the file, but means outside any function/class)
- won't matter in this case, but by defining array inside the function, it's recreated again on each function call - when defined outside the function, it's only defined once (thus a bit more memory efficient) - doesn't matter in this case as the function is only called once, but still good to keep in mind

though, there is also a reason not move the array out of the function as then it's a bit father from where it's used in the code, thus making code a bit harder to follow

Reply:
I think we should leave it inside the function since 
1. It is called once
2. Makes it easier to follow in the code
but I will keep this in mind going forward. thank you for sharing your feedback! 😄